### ISSUE TITLE
Application Error on QA implementation due to date time format

### ISSUE THREAD

**Issue by charissasylim** [rsx-labs/aide-frontend#588]
**Describe the bug**
Application errors in various screens for QA implementation. See Teams QA trail

**Version (please complete the following information):**
 - Version 3.3.3

**Comment by charissasylim** [comment]
@trashvin pending error as of v3.3.5

Rea -
Scenario 1 - Error encountered
Steps:
1. Install Backend Service and CommCell 3.3.5.0 
2. Double click AIDE CommCell Desktop icon
---> no more login error (previous bug fixed)
---> response time: around 1 min before Good morning r.mena@fujitsu.com image displayed
3. Click Assets tab
4. Click Asset Assignment tab
5. Double click Asset ID 2507
**---> error message displayed**

Nina -
•	The error, “An application error was encountered.”, is still occurring whenever I clicked the Attendance module.
- Unable to view resource planner
- Unable to add 3C even if it says that 3C was added
- Unable to input future dated due date in Actions List

Errors related to date format since nina and rea laptop are using dd/mm/yyyy date format.
